enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1158 @@
+
+  Examples:
+  - Adds a finalizer to namespaces. Namespace cannot be deleted without our controller running.
+  - Restricts the label format for objects to X.
+  - Defaults field Y on object kind Z.
+
+Fill in the operational impact of these API Extensions in the "Operational Aspects
+of API Extensions" section.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+In the HCP topology, the HostedCluster and NodePool resources are enhanced to support the change management strategies
+`MaintenanceSchedule` and `Disabled`. 
+
+#### Standalone Clusters
+
+In the Standalone topology, the ClusterVersion and MachineConfigPool resources are enhanced to support the change management strategies
+`MaintenanceSchedule` and `Disabled`. The MachineConfigPool also supports the `Manual` and `Assisted` strategies. 
+
+#### Single-node Deployments or MicroShift
+
+The ClusterVersion operator will honor the change management field just as in a standalone profile. If those profiles
+have a MachineConfigPool, material changes the node could be controlled with a change management policy
+in that resource.
+
+#### OCM Managed Profiles
+OpenShift Cluster Manager (OCM) should expose a user interface allowing users to manage their change management policy.
+Standard Fleet clusters will expose the option to configure the MaintenanceSchedule strategy - including
+only permit and exclude times.
+
+- Service Delivery will reserve the right to disable this strategy for emergency corrective actions.
+- Service Delivery should constrain permit & exclude configurations based on their internal policies. For example, customers may be forced to enable permissive windows which amount to at least 6 hours a month.
+
+### Implementation Details/Notes/Constraints
+
+#### ChangeManagement Stanza
+The change management stanza will be introduced into ClusterVersion and MachineConfigPool (for standalone profiles)
+and HostedCluster and NodePool (for HCP profiles). The structure of the stanza is:
+
+```yaml
+spec:
+  changeManagement:
+    # The active strategy for change management (unless disabled by disabledUntil).
+    strategy: <strategy-name|Disabled|null>
+    
+    # If set to true or a future date, the effective change management strategy is Disabled. Date 
+    # must be RFC3339. 
+    disabledUntil: <bool|date|null>
+    
+    # If set to true or a future date, all strategies other than Disabled are paused. Date 
+    # must be RFC3339. 
+    pausedUntil: <bool|date|null>    
+    
+    # If a strategy needs additional configuration information, it can read a 
+    # key bearing its name in the config stanza.
+    config:
+       <strategy-name>:
+         ...configuration policy for the strategy...
+```
+
+#### MaintenanceSchedule Strategy Configuration
+
+```yaml
+spec:
+  changeManagement:
+    strategy: MaintenanceSchedule
+    config:
+       maintenanceSchedule:
+        # Specifies a reoccurring permissive window. 
+        permit:  
```

Comment:
Is there a way to configure a rule to span days?  I.e. permit start = 10PM UTC on Saturday, end 2AM UTC on Sunday